db/primary_migrate/20231018230656_add_expiration_notice_sent_at_to_usps_confirmation_codes.rb

```diff
@@ -0,0 +1,8 @@
+class AddExpirationNoticeSentAtToUspsConfirmationCodes < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_column :usps_confirmation_codes, :expiration_notice_sent_at, :datetime, precision: nil
+    add_index :usps_confirmation_codes, :expiration_notice_sent_at, algorithm: :concurrently
```

Comment:
Migration to add an indexed column to `usps_confirmation_codes` to track the date/time we sent an expiration notice for a single code.